### ISSUE TITLE
tfb-status don't have enough free space

### ISSUE THREAD

**Issue by joanhey** [TechEmpower/FrameworkBenchmarks#6582]
In the last 2 runs, the latest frameworks fail with:
`E: You don't have enough free space in /var/cache/apt/archives/.`

[yii2](https://tfb-status.techempower.com/unzip/results.2021-05-12-03-16-44-950.zip/results/20210507120805/yii2/build/yii2.log)
[yii2-raw](https://tfb-status.techempower.com/unzip/results.2021-05-12-03-16-44-950.zip/results/20210507120805/yii2-raw/build/yii2-raw.log)
[zend](https://tfb-status.techempower.com/unzip/results.2021-05-12-03-16-44-950.zip/results/20210507120805/zend/build/zend.log)

**Comment by joanhey** [comment]
In every run more frameworks have this problem.

Try to do:
`docker system prune `
in Citrine, to clean old images and containers and regain space.

**Comment by joanhey** [comment]
I need to do it in local sometimes, to remove old images and gain space again.

**Comment by joanhey** [comment]
The problem is worst every time.

Almost all framework that start with letter 'W' fail in the build stage, with `no space left on device` .

- https://tfb-status.techempower.com/unzip/results.2021-11-04-00-34-32-276.zip/results/20211030073108/warp-hasql/build/warp-shared.log

- https://tfb-status.techempower.com/unzip/results.2021-11-04-00-34-32-276.zip/results/20211030073108/webman/build/webman.log
- https://tfb-status.techempower.com/unzip/results.2021-11-04-00-34-32-276.zip/results/20211030073108/workerman-pgsql/build/workerman-pgsql.log
- https://tfb-status.techempower.com/unzip/results.2021-11-04-00-34-32-276.zip/results/20211030073108/workerman/build/workerman.log
- https://tfb-status.techempower.com/unzip/results.2021-11-04-00-34-32-276.zip/results/20211030073108/zio-http/build/zio-http.log
- ...

**Comment by joanhey** [comment]
Now it's failing with frameworks starting with the letter 'u'.

**Comment by joanhey** [comment]
@nbrady-techempower is time to fix this issue.

Every time more frameworks are failing for this issue.
Affect also #6591
https://stackoverflow.com/questions/59139453/repository-is-not-signed-in-docker-build

The toolset must do every  ~200 frameworks `docker system prune --force`

**Comment by joanhey** [comment]
Try a run ordering the frameworks in inverse alphabetical order.
And a lot will work without problems.

**Comment by nbrady-techempower** [comment]
@joanhey This should be resolved now. I've done some manual cleanup during the current run to ensure we get to the end, but the next full run shouldn't have any problems. I'll leave this open until that's confirmed. Thanks.

**Comment by joanhey** [comment]
This run is failing with frameworks starting with the letter 'K'.

**Comment by nbrady-techempower** [comment]
Yeah this may have been from me mucking around manually. Restarting the run from the top with the new toolset changes.

**Comment by joanhey** [comment]
LGTM
This run change from ~130 fw that don't start to 75.
And also fix https://github.com/TechEmpower/FrameworkBenchmarks/issues/6591

The last frameworks start correctly.
